src/Microsoft.TestPlatform.Extensions.BlameDataCollector/PublicAPI/PublicAPI.Shipped.txt

```diff
@@ -81,3 +78,6 @@ Microsoft.TestPlatform.Extensions.BlameDataCollector.XmlReaderWriter.ReadTestSeq
 Microsoft.TestPlatform.Extensions.BlameDataCollector.XmlReaderWriter.WriteTestSequence(System.Collections.Generic.List<System.Guid>! testSequence, System.Collections.Generic.Dictionary<System.Guid, Microsoft.TestPlatform.Extensions.BlameDataCollector.BlameTestObject!>! testObjectDictionary, string! filePath) -> string!
 Microsoft.TestPlatform.Extensions.BlameDataCollector.XmlReaderWriter.XmlReaderWriter(Microsoft.VisualStudio.TestPlatform.Utilities.Helpers.Interfaces.IFileHelper! fileHelper) -> void
 override Microsoft.TestPlatform.Extensions.BlameDataCollector.BlameCollector.Initialize(System.Xml.XmlElement? configurationElement, Microsoft.VisualStudio.TestPlatform.ObjectModel.DataCollection.DataCollectionEvents! events, Microsoft.VisualStudio.TestPlatform.ObjectModel.DataCollection.DataCollectionSink! dataSink, Microsoft.VisualStudio.TestPlatform.ObjectModel.DataCollection.DataCollectionLogger! logger, Microsoft.VisualStudio.TestPlatform.ObjectModel.DataCollection.DataCollectionEnvironmentContext? environmentContext) -> void
+Microsoft.TestPlatform.Extensions.BlameDataCollector.ICrashDumper.GetDumpFiles(bool processCrashed, System.DateTime? testSessionEndTimestamp) -> System.Collections.Generic.IEnumerable<string!>!
```

Comment:
Shall we duplicate APIs instead of doing this binary breaking change?

Reply:
Can we remove from public API if that's only internal then?

Reply:
Can I do that later?

Reply:
Sure, as always, I am just giving some feedback but you are free to ignore them :)

Reply:
ended up reverting all api changes.